chapter3/README.md

```diff
@@ -10,4 +10,5 @@ maxdepth: 1
 ---
 
 astroml_chapter3_Univariate_Distribution_Functions
-```
\ No newline at end of file
+astroml_chapter3_Univariate_Distribution_Functions
```

Comment:
please use the name of the new notebook rather than duplicate the previous one. Thanks!